fe/fe-core/src/test/java/org/apache/doris/analysis/CreateTableStmtTest.java

```diff
@@ -274,4 +274,21 @@ public void testCreateIcebergTable() throws UserException {
                 "\"iceberg.hive.metastore.uris\"  =  \"thrift://127.0.0.1:9087\",\n" +
                 "\"iceberg.table\"  =  \"test\")", stmt.toString());
     }
+
+    @Test
+    public void testCreateHudiTable() throws UserException {
```

Comment:
It's better to add another test case with column definations.

Reply:
have added test case with column definations in https://github.com/apache/incubator-doris/pull/9752